### PR TITLE
AST-3079 - fix:Font size not working in related post.

### PR DESCRIPTION
**Pull request by yssambare12**
**Description** - 
The Related Post Title Font Size does not work in the responsive mode. It seems to be working in the customizer but nothing happens on the front end.

**How to reproduce:**
Enable the related posts
From the design tab, try changing the related posts title font for mobile/tablet devices.
Changes are reflected in the customizer
Publish the changes and check the frontend. Font size is not changed on the front.

**Refer to the screencast:** 
https://d.pr/v/RVvCwK

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: assets/js/unminified/customizer-preview.js [hunk 1]
```diff
@@ -1686,9 +1686,10 @@ function hasWordPressWidgetBlockEditor() {
 	// Related Posts - Customizer preview for Post Title.
 	astra_generate_outside_font_family_css( 'astra-settings[related-posts-title-font-family]', '.ast-related-post-content .entry-header .ast-related-post-title, .ast-related-post-content .entry-header .ast-related-post-title a' );
 	astra_css( 'astra-settings[related-posts-title-font-weight]', 'font-weight', '.ast-related-post-content .entry-header .ast-related-post-title, .ast-related-post-content .entry-header .ast-related-post-title a' );
-	astra_responsive_font_size( 'astra-settings[related-posts-title-font-size]', '.ast-related-post-content .entry-header .ast-related-post-title, .ast-related-post-content .entry-header .ast-related-post-title a' );
+	astra_responsive_font_size( 'astra-settings[related-posts-title-font-size]', '.ast-related-post-content .entry-header .ast-related-post-title, .ast-related-post-content .entry-header .ast-related-post-title a', 'mobile' );
+	astra_responsive_font_size( 'astra-settings[related-posts-title-font-size]', '.ast-related-post-content .entry-header .ast-related-post-title, .ast-related-post-content .entry-header .ast-related-post-title a', 'tablet' );
```
Comment (imnavanath) [inline assets/js/unminified/customizer-preview.js:1690]: Function `astra_responsive_font_size` itself handled responsiveness, no need to add device wise. The preview not applying there reason can be different, please backtrack @yssambare12

File: inc/modules/related-posts/css/dynamic-css.php [hunk 2]
```diff
@@ -91,7 +91,7 @@ function astra_related_posts_css( $dynamic_css ) {
 			/**
 			 * Related Posts - Post Title
 			 */
-			'.ast-related-post-content .entry-header .ast-related-post-title, .ast-related-post-content .entry-header .ast-related-post-title a' => astra_get_font_array_css( astra_get_option( 'related-posts-title-font-family' ), astra_get_option( 'related-posts-title-font-weight' ), $related_post_title_font_size, 'related-posts-title-font-extras', $related_post_text_color ),
+			'.ast-related-post-content .entry-header .ast-related-post-title, .ast-related-post-content .entry-header .ast-related-post-title a' => astra_get_font_array_css( astra_get_option( 'related-posts-title-font-family' ), astra_get_option( 'related-posts-title-font-weight' ), astra_get_option( 'related-posts-title-font-size' ), 'related-posts-title-font-extras', $related_post_text_color ),
```
Comment (imnavanath) [inline inc/modules/related-posts/css/dynamic-css.php:94]: There is already same variable with same astra_get_option call is added, so updated again with astra_get_option any reason for this @yssambare12 ?? https://share.getcloudapp.com/eDurBRyY

**Comment by yssambare12** [comment]
Closing this PR due to lots of conflicts and its continue with updated changes - https://github.com/brainstormforce/astra/pull/5071

File: assets/js/unminified/customizer-preview.js (hunk 1 above)
Comment (yssambare12) [reply assets/js/unminified/customizer-preview.js:1690]: I found the cause and its continue to here https://github.com/brainstormforce/astra/pull/5071

File: inc/modules/related-posts/css/dynamic-css.php (hunk 2 above)
Comment (yssambare12) [reply inc/modules/related-posts/css/dynamic-css.php:94]: I updated the code here - https://github.com/brainstormforce/astra/pull/5071

**Pull request closed without merging**